src/sentry/static/sentry/app/views/eventsV2/relatedEvents.tsx

```diff
@@ -97,12 +98,10 @@ class RelatedEvents extends AsyncComponent<Props> {
           <Card>{t('No related events found.')}</Card>
         ) : (
           events.data.map(item => {
+            const eventSlug = `${item['project.name']}:${item.id}`;
```

Comment:
This slug format is in a bunch of places, should we have a util function so that if the format changes we don't have to hunt down all the places we generate the slug?

Reply:
addressed in https://github.com/getsentry/sentry/pull/15562/commits/e9077510cbe6d3c6a51812b35ea0507224e5bc2d